app.py

```diff
@@ -686,15 +685,9 @@ def edit():
 		treestr = writediscbrackettree(tree, senttok, pretty=True).rstrip()
 		rows = max(5, treestr.count('\n') + 1)
 	else:
-		tree_to_cgel = tree.copy(deep=True)
-		tree_to_cgel = remove_punctuation_nodes(tree_to_cgel)
-		senttok_merged_punct = merge_punctuation(senttok)
-		# writetree requires a string to be passed as its third argument; '1' is a dummy value
-		block = writetree(tree_to_cgel, senttok_merged_punct, '1', 'export', comment='')  #comment='%s %r' % (username, actions))
-		block = io.StringIO(block)
-		treestr = next(load_as_cgel(block))
-		cgel_tree_terminals = handle_punctuation(treestr)
-		treestr.update_terminals(cgel_tree_terminals, gaps=True)
+		senttok = ["-LRB-" if t == "(" else "-RRB-" if t == ")" else t for t in senttok]
```

Comment:
A more extensible solution would be to implement an `escape_token()` function (in case other kinds of escaping become necessary in the future).

Reply:
Actually, if you want to be more specific, maybe `escape_token_for_parser()` (because escaping in the CGEL format will be different)

Reply:
Great, addressed in https://github.com/nschneid/activedop/pull/47/commits/80285bf669d29dfb9550039f6940b06f10bce436